include/mitsuba/core/properties.h

```diff
@@ -54,6 +53,7 @@ class MTS_EXPORT_CORE Properties {
     };
 
     using Float = float;
```

Comment:
I wouldn't change this. It would make the code a bit heavy, and this is also done in other files (e.g. `bitmap.h`, `transform.h`, `spiral.h`, `xml.cpp`, ...)